rotkehlchen/tests/unit/test_savepoints.py

```diff
@@ -0,0 +1,44 @@
+import sqlite3
+
+import pytest
+
+from rotkehlchen.globaldb.handler import GlobalDBHandler
+
+
+def test_savepoints():
```

Comment:
split the test names when you implement the tests I told you. Easier to read when you got small simple unit tests, testing one thing and having explicitly and nicely named names